models/stg_apple_store__sales_subscription_summary.sql

```diff
@@ -29,23 +29,42 @@ final as (
 
     select
         cast(source_relation as {{ dbt.type_string() }}) as source_relation, 
-        cast({{ get_date_from_string( dbt.split_part(string_text='_filename', delimiter_text="'_'", part_number=3)) }} as date) as date_day, 
+        cast(date as date) as date_day,
+        cast(_fivetran_id as {{ dbt.type_string() }}) as _fivetran_id,
+        cast(vendor_number as {{ dbt.type_int() }}) as vendor_number,
+        cast(app_apple_id as {{ dbt.type_int() }}) as app_apple_id,
         cast(app_name as {{ dbt.type_string() }}) as app_name,
-        cast(account_number as {{ dbt.type_bigint() }}) as account_id,
-        cast(country as {{ dbt.type_string() }}) as country,
-        cast(case
-            when replace(state, ' ', '') = '' then cast(null as {{ dbt.type_string() }}) else state
-        end as {{ dbt.type_string() }}) as state,
         cast(subscription_name as {{ dbt.type_string() }}) as subscription_name,
-        cast(case 
-            when lower(device) like 'ipod%' then 'iPod' else device
-        end as {{ dbt.type_string() }}) as device,
-        sum(cast(active_free_trial_introductory_offer_subscriptions as {{ dbt.type_bigint() }})) as active_free_trial_introductory_offer_subscriptions,
-        sum(cast(active_pay_as_you_go_introductory_offer_subscriptions as {{ dbt.type_bigint() }})) as active_pay_as_you_go_introductory_offer_subscriptions,
-        sum(cast(active_pay_up_front_introductory_offer_subscriptions as {{ dbt.type_bigint() }})) as active_pay_up_front_introductory_offer_subscriptions,
-        sum(cast(active_standard_price_subscriptions as {{ dbt.type_bigint() }})) as active_standard_price_subscriptions
+        cast(subscription_apple_id as {{ dbt.type_int() }}) as subscription_apple_id,
+        cast(subscription_group_id as {{ dbt.type_int() }}) as subscription_group_id,
+        cast(standard_subscription_duration as {{ dbt.type_string() }}) as standard_subscription_duration,
+        cast(customer_price as {{ dbt.type_float() }}) as customer_price,
+        cast(customer_currency as {{ dbt.type_string() }}) as customer_currency,
+        cast(developer_proceeds as {{ dbt.type_float() }}) as developer_proceeds,
+        cast(proceeds_currency as {{ dbt.type_string() }}) as proceeds_currency,
+        cast(preserved_pricing as {{ dbt.type_string() }}) as preserved_pricing,
+        cast(proceeds_reason as {{ dbt.type_string() }}) as proceeds_reason,
+        cast(subscription_offer_name as {{ dbt.type_string() }}) as subscription_offer_name,
+        cast(promotional_offer_id as {{ dbt.type_string() }}) as promotional_offer_id,
+        cast(state as {{ dbt.type_string() }}) as state,
```

Comment:
Same question regarding if you confirm the `state` and `quantity` case statements are no longer necessary in this schema?

Reply:
Same comment as above